documentation/freshdesk.md

```diff
@@ -0,0 +1,34 @@
+# Freshdesk
+
+Freshdesk is the customer service software used by GC Digital Talent for creating support tickets.
+
+## Account
+
```

Comment:
Maybe a small mention that there is no shared test environment.  For testing, either sign up for a new account or send test messages to the prod mailbox.

Reply:
See d9ae6e65cac8b7d20d2ba9aeb427cb56645d43c4.